config/v0.3.0/propagation.go

```diff
@@ -0,0 +1,41 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package config // import "go.opentelemetry.io/contrib/config/v0.3.0"
+
+import (
+	"errors"
+
+	"go.opentelemetry.io/contrib/propagators/autoprop"
+	"go.opentelemetry.io/otel/propagation"
+)
+
+var (
+	errInvalidPropagatorEmpty = errors.New("invalid propagator name: empty")
+	errInvalidPropagatorNil   = errors.New("invalid propagator name: nil")
+)
+
+func propagator(cfg configOptions) (propagation.TextMapPropagator, error) {
+	if cfg.opentelemetryConfig.Propagator == nil {
+		return autoprop.NewTextMapPropagator(), nil
+	}
+
+	n := len(cfg.opentelemetryConfig.Propagator.Composite)
+	if n == 0 {
+		return autoprop.NewTextMapPropagator(), nil
+	}
+
+	names := make([]string, 0, n)
+	for _, name := range cfg.opentelemetryConfig.Propagator.Composite {
+		if name == nil {
+			return nil, errInvalidPropagatorNil
+		}
+		if *name == "" {
+			return nil, errInvalidPropagatorEmpty
+		}
```

Comment:
I do not see anything in the schema that disallows using empty name. See: https://github.com/open-telemetry/opentelemetry-configuration/blob/main/schema/propagator.json

`autoprop` supports empty names (similarly to `database/sql`).

Reply:
i opened an [issue](https://github.com/open-telemetry/opentelemetry-configuration/issues/172) in the configuration repo to discuss this, i submitted a PR to propose enforcing a minimum length on propagator configuration